### ISSUE TITLE
How

### ISSUE THREAD

**Issue by fandy** [cyanharlow/purecss-francine#2]
You should totally write about how you built this!

**Comment by WagnerMoreira** [comment]
and how much time it took to built too

**Comment by vinnyA3** [comment]
This is quite the work of art, that's for sure.  I would love to hear your process.

**Comment by jepster** [comment]
I am also interested in the approach. It looks unbelievable. It's hard to believe, that this graphics are not just exported from adobe illustrator in svg format. :)

**Comment by jesraygarciano** [comment]
Where should we start when we're making our own :)

**Comment by Sm00g15** [comment]
So good OMG! great job!

**Comment by guylhermetabosa** [comment]
Awesome Job, I totally agree with the OP.

**Comment by podrivo** [comment]
Don't forget to livestream it and make a speed video.

**Comment by alessandro308** [comment]
@Sm00g15 That's not true. The link is done by drawing in a canvas, it is completely different approach and it is easier to do a drawing with a canvas. She creates the drawing with pure HTML elements, <div> with custom class!!
